Modix.Bot/Behaviors/PromotionPollBehavior/PromotionDialogueBehavior.cs

```diff
@@ -0,0 +1,161 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Linq;
+using Discord;
+using Discord.WebSocket;
+using Microsoft.Extensions.Caching.Memory;
+using Modix.Common.Messaging;
+using Modix.Data.Models.Core;
+using Modix.Data.Models.Promotions;
+using Modix.Services.Core;
+using Modix.Services.Images;
+using Modix.Services.Promotions;
+using Modix.Services.Utilities;
+
+namespace Modix.Bot.Behaviors
+{
+    public class PromotionDialogueBehavior :
+        INotificationHandler<ReactionAddedNotification>,
+        INotificationHandler<PromotionActionCreatedNotification>
+    {
+        private readonly DateTime _utcNow = DateTime.UtcNow;
```

Comment:
Nitpick: `private`s belong at the bottom of the class, and don't need line spacing

Reply:
Really? ive never seen private variables at the bottom in a project and even in the majority of modixs code base they are at the top

Reply:
Not in anything I've written. :P

My line-of-thought is that I could give a fuck about what the `private` members of a `class` are, 95% of the time. The first things I should see within a file are the things I'm likely to care about by opening it: what is its API, and what do I need to do to get an instance of it. Yes, dependencies are important, and something I'm likely to care about, even if I'm not going to be changing that class, but that's why the constructor should be at, or near, the top. `private` fields are not the same as the class's dependencies.

We don't really have an official standard/style either way within MODiX, so you're free to do it however you like. If it comes to a vote, though, that's my vote, and my reasoning behind it.